jdkhttp/Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 RUN microdnf install java-1.8.0-openjdk-headless && microdnf clean all
+ENV HOME /deployments
 COPY target/*-jdkhttp-fat.jar /deployments/app.jar
-COPY ./scripts/jboss-settings.xml /root/.m2/settings.xml
+COPY ./scripts/jboss-settings.xml /deployments/.m2/settings.xml
 COPY ./scripts/logging.sh /deployments/logging.sh
 COPY ./scripts/maven.sh /deployments/maven.sh
 COPY ./scripts/start.sh /deployments/start.sh
+RUN chmod 775 /deployments/logging.sh
```

Comment:
Found out that it will be necessary to also relax permissions on deployment folder and some items inside of it. Something like:
RUN chmod 777 /deployments /deployments/.m2
RUN chmod 666 /deployments/.m2/settings.xml

Reply:
ok